runtime-testsuite/resources/org/antlr/v4/test/runtime/new_descriptors/FullContextParsing/AmbigYieldsCtxSensitiveDFA.txt

```diff
@@ -0,0 +1,27 @@
+[type]
+Parser
+
+[grammar]
+grammar T;
+s @after {<DumpDFA()>}
+   : ID | ID {} ;
+ID : 'a'..'z'+;
+WS : (' '|'\t'|'\n')+ -> skip ;
+
+[start]
+s
+
+[input]
+abc
+
+[output]
+Decision 0:
+s0-ID->:s1^=>1
+
+[errors]
+"""line 1:0 reportAttemptingFullContext d=0 (s), input='abc'
+"""
```

Comment:
`"""` can be removed for all `[errors]`. It does not matter how many line breaks in errors, they can be trimmed.

Reply:
It's tricky with the new lines because sometimes we want to ignore white space and sometimes we don't. Probably for error messages you are right but let's start with a general translation and then figure out how to make it simpler. Although it might not be necessary since we don't really create that many new tests./